Makefile

```diff
@@ -1,7 +1,9 @@
-.PHONY: all install install-dependencies install-tools test test-verbose
+.PHONY: all install install-dependencies install-tools test test-full test-verbose
```

Comment:
That is and should be test-verbose. You are thinking about the rename of "test-throughout" to "test-full".